setup.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+setup(
+    name = 'HTSOHM',
+    version = '0.1.0',
+    description = 'High-throughput Screening of Hypothetical Materials',
+    author = 'A Reino Kaija',
+    author_email = 'ark111@pitt.edu',
+    url = 'https://github.com/akaija/HTSOHM-dev',
+    packages = ['htsohm', 'htsohm.db', 'htsohm.simulation']
+)
```

Comment:
If you're adding a setup.py, you should consider adding your binary to it too so it can be run from any directory.